content/docs/tutorials/aws/ec2-webserver.md

```diff
@@ -113,7 +113,7 @@ aws:region: (us-east-1)
     server = ec2.Instance('webserver-www',
         instance_type=size,
         security_groups=[group.name], # reference security group from above
-        ami=get_linux_ami(size))
+        ami='ami-0ff8a91507f77f867')
```

Comment:
I don't think we should hardcode this - what region is this for? I would suggest we use the getAmi datasource to get the value instead? Just my $0.02